docs/installing.md

```diff
@@ -1,4 +1,17 @@
-# OS X fast track
+
+# Prerequisites
+
+## Operating system
+
+gcsfuse has been tested successfully with the following operating systems:
+
+*   Linux (minimum kernel version 3.10)
+*   OS X (minimum version 10.10.2)
+
+It may or may not work correctly with other operating systems and older versions.
+
+___
+### OS X fast track
```

Comment:
Thanks for doing this, but it doesn't make sense to nest the "fast track" instructions under "Prerequisites", since they take care of the prerequisites for you. If the problem is that the operating systems aren't up front, I suggest making "Operating Systems" a top-level section, then the fast track section, then prerequisites.